src/components/display/PersonCard/PersonCard.stories.tsx

```diff
@@ -40,6 +40,17 @@ export const FavouriteCharacter: StoryObj<typeof meta> = {
   },
 };
 
+export const FavouriteCharacterWithEdit: StoryObj<typeof meta> = {
+  args: {
+    name: "Leia Organa",
+    gender: "Female",
+    planet: "Alderaan",
+    height: "120",
+    onDelete: () => {},
+    onEdit: () => {},
+  },
+};
+
```

Comment:
Added new story to test how delete and edit buttons look like on favourite character card.